terraform/environments/electronic-monitoring-data/modules/lambdas/main.tf

```diff
@@ -1,5 +1,6 @@
 locals {
   use_vpc_config = !(var.security_group_ids == null || var.subnet_ids == null)
+  function_uri = var.function_tag != null ? var.function_tag : (var.is_image ? "${var.function_name}-${var.production_dev}" : "")
```

Comment:
My terraform is a little rusty to understand this line, hah

Reply:
haha this will need cleaning up but effectively it means if you add `function_tag` in the arguments that will be the `function_uri`, if not and `is_image` is true it will be `"${var.function_name}-${var.production_dev}"` else it will be an empty string